scripts/test_examples.sh

```diff
@@ -201,44 +201,457 @@ stopKafka
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '6', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '7', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '8', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '10', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '11', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '12', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '13', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '14', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '15', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '16', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '17', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '18', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '19', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '20', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '21', key: 'proto-key', 
+Sending 0 sync Kafka notifications (protobuf) ...
+Sending 0 async Kafka notifications (protobuf) ...
+")
+
+
+
+cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf --messageCount 0 -offsetMsg=oldest"
+testOutput "${cmd}" "${expected}"
+
+
+# Let us test - in example no new messages generated - wrong value of parameter offsetMsg
+expected=("Invalid partition -1 - defaulting to 0
+Error loading core: plugin KafkaExample: Init error 'incorrect sync offset value wronginput
+")
+
+
+cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf --messageCount 0 -offsetMsg=wronginput"
+testOutput "${cmd}" "${expected}"
+
+
 stopKafka
 
 #### Kafka-plugin hash-partitioner #######################################
 
 startKafka
 
-expected=("Sending 10 sync Kafka notifications
-Sending 10 async Kafka notifications
-Received Kafka Message, topic 'example-sync-topic'
-Async message successfully delivered, topic 'example-async-topic'
-Received async Kafka Message, topic 'example-async-topic'
-Sync watcher closed
-Async watcher closed
+# Let us test parameters display
```

Comment:
I'm thinking whether this have some added value. The goal of example testing is to verify kafka functionality. This has nothing to do with it.

Reply:
I removed all the parameters tests instances...